module/solutions/helper.js

```diff
@@ -5,1133 +5,1013 @@
  * Description : Solution related helper functionality.
  */
 
-const { ObjectId } = require("mongodb");
+const { ObjectId } = require('mongodb')
 
 // Dependencies
 
-const solutionsQueries = require(DB_QUERY_BASE_PATH + "/solutions");
-const programsHelper = require(MODULES_BASE_PATH + "/programs/helper")
-const programQueries = require(DB_QUERY_BASE_PATH + "/programs")
+const solutionsQueries = require(DB_QUERY_BASE_PATH + '/solutions')
```

Comment:
@MallanagoudaB What is the change in this file or is it just formatting?

Reply:
i think it was made by prajwal